kvmagent/kvmagent/plugins/zrm_plugin.py

```
@@ -0,0 +1,1234 @@
+                    # A concluded mirror job means the job has finished — possibly
+                    # with an error (e.g. NBD target disconnected). We must report
+                    # this to the ZRM so it can trigger recovery, and dismiss the
+                    # job to clean up QEMU state (same as _start_mirrors_for_zr).
+                    concluded_count += 1
+                    err_text = job.get("error") or "no error detail"
+                    concluded_errors.append({"device": device, "error": str(err_text)})
+                    try:
+                        qmp.execute_qmp_command(vm_uuid, "block-job-dismiss",
+                                                raise_exception=False, id=device)
+                    except Exception:
+                        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)